nox/_options.py

```diff
@@ -146,6 +146,30 @@ def _envdir_merge_func(
     return command_args.envdir or noxfile_args.envdir or ".nox"
 
 
+def _reuse_venv_merge_func(
```

Comment:
question (non-blocking): Would it be possible to use `Literal["no", "yes", "never", "always"]` to improve type checking/IDE behavior?

Is there any way to make it so that IDEs/mypy can type check a user's input to `reuse_venv=` before running Nox?

Reply:
I didn't include it since we're still supporting 3.7 in the tests. Is there an alternative here?

Reply:
One option could be [`typing-extensions`](https://pypi.org/project/typing-extensions/):

```py
from typing_extensions import Literal
```

Reply:
Good catch, didn't notice `typing_extensions` was already installed

Reply:
Yeah, this should have us covered

https://github.com/wntrblm/nox/blob/274df9be1b977d7f4a6323f2dcb7ace6e49bf0bd/pyproject.toml#L47

Reply:
See c9d6ad97c41ec02795552fb873330897527de40a